src/cljs/rems/administration/components.cljs

```diff
@@ -35,31 +36,39 @@
   [:div {:class "invalid-feedback"}
    (when error (text-format error label))])
 
-(defn input-field [{:keys [keys label placeholder context type normalizer readonly]}]
+(defn input-field [{:keys [keys label placeholder context type normalizer readonly inline?]}]
```

Comment:
not a fan of calling it inline, it is easily confused with a CSS property but do not have a strong opinion

Reply:
We use `inline?` in rems.atoms/info-field and `inline-info-field` in rems.administration.components, that's where I copied it from. It's not ideal I agree.

Reply:
This is where it comes from:
https://getbootstrap.com/docs/4.0/components/forms/#inline

Reply:
I see, in any case, I think this is a great feature and ok to merge